key_management/src/wallet_helpers.rs

```diff
@@ -53,11 +54,13 @@ pub fn sign(sig_type: SignatureType, private_key: &[u8], msg: &[u8]) -> Result<S
         SignatureType::Secp256 => {
```

Comment:
nit but I didn't notice this in previous review, can you change this:
```suggestion
        SignatureType::Secp256k1 => {
```